bot/seasons/easter/__init__.py

```diff
@@ -3,14 +3,23 @@
 
 class Easter(SeasonBase):
     """
-    Easter is a beautiful time of the year often celebrated after the first Full Moon of the new spring season.
-    This time is quite beautiful due to the colorful flowers coming out to greet us. So. let's greet Spring
-    in an Easter celebration of contributions.
+    To celebrate, we have a lovely new colourful server icon. Thanks to <@140605665772175361> for
```

Comment:
Should this really be the first thing in the announcement? Sure, it will be more effective here in suppressing sever icon questions, but I feel it takes away focus from the rest of the announcement.

Reply:
Where do you propose? Under `flowers coming out to greet us.`?

Reply:
Yeah. It would actually transition nicely into the next paragraph there.